### PR TITLE
[cmake][windows] Don't install system DLLs

### PR DESCRIPTION
**Pull request by bellenot**
Fix Jira issue ROOT-10926 https://its.cern.ch/jira/browse/ROOT-10926

### CODE REVIEW

**Comment by guitargeek** [comment]
There is a comment just above the line you enclose with the `if`:
```txt
Package up needed system libraries - only for WIN32?
```
Does this mean the line can be removed entirely?

**Comment by bellenot** [comment]
> There is a comment just above the line you enclose with the `if`:
> 
> ```
> Package up needed system libraries - only for WIN32?
> ```
> 
> Seems it was only intended for Windows to begin with. Does this mean the line can be removed entirely?

I guess so, but since I don't know if that affects other platforms I prefer not removing it for the time being

File: cmake/modules/RootCPack.cmake [hunk 1]
```diff
@@ -12,7 +12,9 @@
 #---------------------------------------------------------------------------------------------------
 # Package up needed system libraries - only for WIN32?
```
Comment (ferdymercury) [inline cmake/modules/RootCPack.cmake:13]:
```suggestion
# Package up needed system libraries - except for WIN32
```

**Review by dpiparo — APPROVED** [review]
Thanks a lot: LGTM!

**Pull request merged**